elixir/apps/domain/lib/domain/config/definitions.ex

```diff
@@ -432,6 +433,13 @@ defmodule Domain.Config.Definitions do
     changeset: {Domain.Config.Configuration.ClientsUpstreamDNS, :changeset, []}
   )
 
+  @doc """
+  Delay time in milliseconds to wait before redirecting client on the sign in success page.
```

Comment:
Milliseconds or seconds?

Reply:
Looks like you might have already figured it out, but yeah, it's milliseconds.